src/flask_debugtoolbar/__init__.py

```diff
@@ -131,6 +127,7 @@ def _default_config(self, app):
                 'flask_debugtoolbar.panels.profiler.ProfilerDebugPanel',
                 'flask_debugtoolbar.panels.g.GDebugPanel',
             ),
+            'SQLALCHEMY_RECORD_QUERIES': app.debug,
```

Comment:
Thanks. Should this be moved to the init function of the extension or is a `before_request` a standard way to do this?

Reply:
The highlighted line is currently in `DebugToolbarExtension._default_config()` which gets called by `DebugToolbarExtension.init_app()`. You mean it should be directly set in `DebugToolbarExtension.init_app()` instead of `DebugToolbarExtension._default_config()`?
 
For the mentioned alternative, do you mean [`flask.Flask.before_request`](https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.before_request)? That would mean it is being run before every request, that is probably not what is needed?

Reply:
Oh we're good, sorry. GitHub presented the diff in a way that looked like it was inside of a `before_request`. When viewing the whole file it's good to go.